### PR TITLE
error: Text error message for non 200 form submission

### LINKED ISSUE #313: Text error message for non 200 form submission

### PR DESCRIPTION
**Pull request by Pareesh**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #313 #311 

Test URLs:
- Before: https://main--mammotome--hlxsites.hlx.page/us/en/
- After: https://313-mammotomoe--mammotome--hlxsites.hlx.page/us/en/

### CODE REVIEW

File: blocks/form/form.js [hunk 1]
```diff
@@ -169,6 +182,7 @@ function createSubmit(fd) {
     });
     obs.observe(wrapper);
   }
+  wrapper.appendChild(document.createElement('ul')); // for error messages
```
Comment (jalagari) [inline blocks/form/form.js:185]: Why do we need UL for displaying single error message?

Comment (Pareesh) [reply blocks/form/form.js:185]: I am using the same css to display the error message, which we are using when we display any validation error in attachment component. UL can help show a list of messages, currently we have one(unsuccessful form submission or non 200) but in future if someone wants to show validation errors as well same ul can be used.

Comment (vdua) [reply blocks/form/form.js:185]: > UL can help show a list of messages, currently we have one(unsuccessful form submission or non 200) but in future if someone wants to show validation errors as well same ul can be used.

I think we can update that to UL when required. Right now I would say we use a single div to show the error.

Comment (Pareesh) [reply blocks/form/form.js:185]: ok, I will update the PR.

**Comment by vdua** [comment]
LGTM

**Review by jalagari — APPROVED** [review]

**Pull request merged**